onnxruntime/contrib_ops/rocm/bert/attention_impl.cu

```diff
@@ -78,6 +78,89 @@ inline int3 Get2DMaskStrides(int total_sequence_length) {
   return {total_sequence_length, 0, 1};
 }
 
+Status ClassifyAttentionMode(
+    const std::string& op,
+    RocmAttentionParameters* attn,
+    const std::vector<const Tensor*> qkv,
+    const std::vector<const Tensor*> past,
+    const std::vector<Tensor*> present) {
+  size_t num_qkv = std::count_if(qkv.cbegin(), qkv.cend(), [](auto it) { return it != nullptr; });
```

Comment:
> const std::vector<const Tensor*> qkv,
    const std::vector<const Tensor*> past,
    const std::vector<Tensor*> present
    
   Should these all be const-refs, rather than const-vals?